hw03_barkovskan.py

```diff
@@ -0,0 +1,41 @@
+import turtle
+
+def drawsquare(bubbles, size):
+    """Draw a square using turtle."""
+    for _ in range(4):
+        bubbles.forward(size)
+        bubbles.left(90)
+
+def draw_row(bubbles, color1, color2):
+    """Draw a row of squares with alternating colors."""
+    for _ in range(8):
+        if _ % 2 == 0:
```

Comment:
The underscore variable (`_`) implies it's a temporary variable that you have no intention on using directly. Yet, you used it directly here in the conditional. This indicates you should have used an actual variable name that has meaning, e..g., `for cell in range(8):`